doc/nrf/libraries/others/event_manager_proxy.rst

```diff
@@ -0,0 +1,158 @@
+.. _event_manager_proxy:
+
+Event Manager proxy
```

Comment:
Let's name it Event Manager Proxy
s/proxy/Proxy

So name consists of three words

Reply:
ok